metaflow/event_logger.py

```diff
@@ -17,17 +20,32 @@ def terminate(self):
 
 
 class EventLogger(NullEventLogger):
-    def __init__(self, logger_type):
-        # type: (str) -> None
+    def __init__(self, logger_type, env, **kwargs):
+        # type: (str, str, str) -> None
         self.sidecar_process = None
         self.logger_type = logger_type
+        self._context = {"env": env.get_environment_info()}
```

Comment:
Can't this be done in the constructor for `KeyStoneLogger`?

Reply:
of course. I was just trying to be consistent with the monitor. I am fine either doing both in subclasses or both in the "base classes"

Reply:
I have refactored all this.